lib/model_patches.rb

```diff
@@ -14,4 +15,29 @@
   #     "If you uncomment this line, this text will appear as default text in every message"
   #   end
   # end
+
+  User.class_eval do
+    def self.province_name_options
+      province_names = if FastGettext.locale == "nl_BE"
+        [
+          'Antwerpen',
+          'Brussel',
+          'Limburg',
+          'Oost-Vlaanderen',
+          'West-Vlaanderen',
+          'Vlaams Brabant',
+          'Andere'
+        ]
+      else
+        [
+          'Brabant Wallon',
+          'Bruxelles',
+          'Hainaut',
+          'Liège',
+          'Luxembourg',
+          'Namur Autre'
```

Comment:
That feels very much like a typo from me, I will check against the original email